firestore/integration_test_internal/src/type_test.cc

```diff
@@ -1,3 +1,5 @@
+// Copyright 2021 Google LLC
+
 #include "firebase/firestore.h"
 #include "firestore_integration_test.h"
 
```

Comment:
Lint: `Include the directory when naming .h files `<hidden value="cpplint-file-comment"></hidden>